### ISSUE TITLE
False positive not-an-iterable for pandas.DataFrame.columns

### ISSUE THREAD

**Issue by sam-s** [PyCQA/pylint#2962]
### Steps to reproduce
run `pylint` on file
```
import pandas as pd
df = pd.DataFrame({"a":range(1,11),"b":range(11,21)})
print(sum([len(c) for c in df.columns]))
```

### Current behavior
```
error (E1133, not-an-iterable, ) Non-iterable value df.columns is used in an iterating context
```

### Expected behavior

no output

### pylint --version output
```
pylint 2.3.1
astroid 2.2.5
Python 3.7.3 (default, Mar 27 2019, 09:23:15) 
[Clang 10.0.1 (clang-1001.0.46.3)]
```

**Comment by PCManticore** [comment]
Thank you, I can reproduce the issue.

**Comment by rcasiano8** [comment]
Can confirm this too. It's happening to me on VSCode with pylint.
![image](https://user-images.githubusercontent.com/47330420/65176406-b8761380-da22-11e9-9da0-b63b6bca5c88.png)

**Comment by kzhui125** [comment]
I meet this issue too.

**Comment by Moonshile** [comment]
This issue is bothering for over a year. Is there any plan to resolve it?

**Comment by Pierre-Sassoulas** [comment]
@Moonshile Apparently no one was bothered enough to open a pull request yet.  But you can bring the change you want to see in the world if you want 😉

**Comment by clavedeluna** [comment]
This is no longer an issue!
